ocaml/libs/tracing/tracing_export.ml

```diff
@@ -0,0 +1,346 @@
+(*
+ * Copyright (C) 2024 Cloud Software Group
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+module D = Debug.Make (struct let name = "tracing-export" end)
+
+module Delay = Xapi_stdext_threads.Threadext.Delay
+open D
+open Tracing
+
+let ( let@ ) f x = f x
+
+module Export = struct
+  let export_interval = ref 30.
```

Comment:
This needs a comment what unit this has.

Reply:
`tracing_export.ml` is the previous `Tracing.Export` module with some small modification. I will address your suggestions as a separate commit :)

Reply:
Intervals should usually be encoded as spans of monotonic time. Fortunately we have such library: `Mtime.Span.(30 * s)`

It may not be possible to immediately use these because an API doesn't let you or it's quite a big change. That's fine let me know and I'll add a note so I can change it in a PR that I'm working on.